packages/pwa-kit-runtime/jest.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     ...base,
     coverageThreshold: {
         global: {
-            branches: 92.5,
+            branches: 92.4,
```

Comment:
This is purely due to how the calculation changes for multiple files instead of one.